packages/crayons-icon/package.json

```diff
@@ -53,5 +53,8 @@
     "json-to-pretty-yaml": "^1.2.2",
     "meow": "^9.0.0",
     "svgo": "^1.3.2"
+  },
+  "devDependencies": {
+    "@types/estree": "^0.0.51"
```

Comment:
do we need this dependency here ?

Reply:
Removed.

Reply:
I still see this dependency in crayons-icon package.json. Can you please check ?

Reply:
Yes, removed now.